Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -46,6 +46,7 @@ stages:
       AppImageName: $(AppImageName)
       ContainerRegistryAdminUser: $(ContainerRegistryAdminUser)
       ContainerRegistryPassword: $(ContainerRegistryPassword)
+      SolutionBaseName: ${{ variables.SolutionBaseName }}
```

Comment:
${{ variables.SolutionBaseName }} isn't defined anywhere,  so defaults to an empty string

Reply:
It's at the top of the file.  It's not in the change set because I'd stupidly hardcoded it in some places so these changes are just sorting that out